src/test/kotlin/uk/gov/dwp/dataworks/controllers/ConnectionControllerTest.kt

```diff
@@ -56,6 +64,8 @@ class ConnectionControllerTest {
     fun `401 returned when bad token`() {
         whenever(authService.validate(any())).thenThrow(JWTVerificationException(""))
         mvc.perform(post("/connect")
+                .content("{}")
```

Comment:
This should be valid contents that the API will accept, otherwise it is likely to fail to parse the request and you will get a bad response code that way, instead of from the validation failing.
I'd suggest we check a portion of the error message to ensure the reason a `401` is being returned is due to the validation.